index.html

```diff
@@ -0,0 +1,273 @@
+                </div>
+                <div class="customer_two">
+                    <span>
+                        <img src="customer_2.png" alt="customer">
+                        <h3>Yessica Christy</h3>
+                        <p><br>Shanxi, China</p>
+                        <h6>4.5</h6>
+                        <svg class="icon_star"><use xlink:href="#icon-ant-design_star-filled"></use></svg>
+                    </span>
+                    <article>“I like it because I like to travel far and still can connect with high speed.”</article>
+                </div>
+                <div class="customer_three">
+                    <span>
+                        <img src="customer_3.png" alt="customer">
+                        <h3>Kim Young Jou</h3>
+                        <p><br>Seoul, South Korea</p>
+                        <h6>4.5</h6>
+                        <svg class="icon_star"><use xlink:href="#icon-ant-design_star-filled"></use></svg>
+                    </span>
+                    <article>“This is very unusual for my business that currently requires a virtual private network that has high security.”</article>
+                </div>
+                <div class="bullet">
+                    <div class="union"></div>
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                </div>
+                <button><span class="material-symbols-outlined">arrow_left_alt</span></button>
+                <button><span class="material-symbols-outlined">arrow_right_alt</span></button>
+            </div>
+        </section>
+        <section>
+            <!-- subscribe -->
+            <div class="subscribe">
+                <h1>Subscribe Now for Get Special Features!</h1>
+                <article>Let's subscribe with us and find the fun.</article>
+                <button>Subscribe Now</button>
+            </div>
+            <img src="Background.png" class="background"/>
+        </section>
+    </main>
+    <footer>
+        <div class="footer">
+            <div class="logo">
+                <svg class="icon icon-Vector"><use xlink:href="#icon-Vector"></use></svg>
+                <h3>LaslesVPN</h3>
+            </div>
+            <article>LaslesVPN is a private virtual network that has unique features and has high security.</article>
+            <div class="social">
+                <span>
+                    <svg class="icon-Facebook"><use xlink:href="#icon-Facebook-icon"></use></svg>
+                </span>
+                <span>
+                    <svg class="icon-Twitter"><use xlink:href="#icon-Twitter"></use></svg>
+                </span>
+                <span>
+                    <svg class="icon icon-Instagram"><use xlink:href="#icon-Instagram"></use></svg>
+                </span>
+                <p>©2020LaslesVPN</p>
+            </div>
+        </div>
+        <nav class="navigation">
+            <ul>
+                <p><b>Product</b></p>
```

Comment:
можно просто b без p

```suggestion
                <b>Product</b>
```